js/gulp/package-task.js

```diff
@@ -46,14 +46,19 @@ const createMainPackageJson = (target, format) => (orig) => ({
     ...createTypeScriptPackageJson(target, format)(orig),
     bin: orig.bin,
     name: npmPkgName,
-    main: `${mainExport}.node`,
-    browser: `${mainExport}.dom`,
-    module: `${mainExport}.dom.mjs`,
+    type: 'commonjs',
+    main: `${mainExport}.node.js`,
+    module: `${mainExport}.dom.js`,
+    browser: `${mainExport}.dom.js`,
```

Comment:
```suggestion
    browser: `${mainExport}.dom`,
```

This way bundlers can pick up mjs files more easily, no?